.gitignore

```diff
@@ -1 +1,3 @@
 bin
+
+cert-manager-istio-csr/
```

Comment:
are you sure we want to ignore that directory which has the submodule repo?
For 1.14, likely this directory shouldn't even exist.

Reply:
Istio-csr is for 1.15, but submodules, when we switch branches will have empty dir, if not created. And to avoid confusing users added to ignore list.

Reply:
git automatically ignores empty directories, but this is a problem only when you checkout release-1.15 first and come back to release-1.14 later, wherein the cert-manager-istio-csr directory is left non-empty with the old contents.

`git clean -fd` also doesn't clean the traces of it 🧐

Reply:
it cleans, but everytime we checkout a new branch and come back to the branch where the submodule is not added, we see that dir.